agent/credentials/providers/rotating_shared_credentials_provider.go

```diff
@@ -15,13 +15,16 @@ package providers
 
 import (
 	"fmt"
+	"os"
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/cihub/seelog"
 )
 
 const (
+	EXTERNAL_CREDENTIAL_PROFILE_ENV_VAR = "ECS_EXTERNAL_CREDENTIAL_PROFILE"
```

Comment:
I think this may also be a valid option for EC2 customers. if so, the env var could have a generic name like `ECS_CREDENTIAL_PROFILE`. wdyt?

Reply:
how do you like `ECS_ALTERNATE_CREDENTIAL_PROFILE` ?
I want to highlight that it's an override of the default profile.

Reply:
sounds good